### PR TITLE
4/4 move `pages` to `src/pages`

### PR DESCRIPTION
**Pull request by dimaMachina**
to simplify review, just review last commit 😆

### CODE REVIEW

File: website/src/contracts.tsx [hunk 1]
```diff
@@ -17,19 +17,17 @@ export function ProtocolContractsTable({ networkId }: { networkId: number }) {
     <Table>
       <tbody>
         <tr>
-          <th>{'Contract'}</th>
-          <th>{'Address'}</th>
+          <th>Contract</th>
+          <th>Address</th>
```
Comment (benface) [inline website/src/contracts.tsx:21]: These should probably be translated, adding myself a TODO for that.

**Review by benface — APPROVED** [review]
Thank you! 🙇‍♂️

File: website/src/contracts.tsx (hunk 1 above)
Comment (dimaMachina) [reply website/src/contracts.tsx:21]: want to rollback?

Comment (benface) [reply website/src/contracts.tsx:21]: No no, as it is now, it makes no sense to use interpolation. This is a good change. :P

**Pull request merged**